src/extensions/askUserForParam.ts

```diff
@@ -42,16 +41,20 @@ export function checkResponse(response: string, spec: Param): boolean {
     }
   }
 
-  // Return false if at least one of the responses is not a valid option
   if (spec.type === ParamType.MULTISELECT || spec.type === ParamType.SELECT) {
-    return !_.some(responses, (r) => {
-      if (!extensionOptionToValue(r, spec.options as ParamOption[])) {
-        utils.logWarning(`${r} is not a valid option for ${spec.param}.`);
+    // Return true if every response is valid.
+    return _.every(responses, (r) => {
```

Comment:
If multiple responses are invalid, it would be great to log a warning for each. I did a little testing ,and it looks like _.every shortcuts and stops if it gets a false, but _.some doesn't (https://repl.it/repls/DearestBarrenMineral).

Switch this back to !_.some?

Reply:
Nice catch! I switched to forEach to make it more readable